build.gradle

```diff
@@ -38,7 +38,7 @@ buildscript {
         kmnid_version = "0.3.2"
         kethereum_version = "0.75.1"
         tweetnacl_k_version = "0.0.1"
-        did_jwt_version = "0.2.0"
+        did_jwt_version = "36f91a7562"//"0.3.0"
```

Comment:
version 0.3.1 will be available with the required changes after [this PR](https://github.com/uport-project/kotlin-did-jwt/pull/17) gets approved

Reply:
updated to 0.3.1